api/renter_test.go

```diff
@@ -9,16 +9,19 @@ import (
 	"testing"
 	"time"
 
+	"bytes"
+	"fmt"
 	"github.com/NebulousLabs/Sia/build"
 	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/Sia/modules/renter"
 	"github.com/NebulousLabs/Sia/modules/renter/contractor"
 	"github.com/NebulousLabs/Sia/types"
 	"github.com/NebulousLabs/fastrand"
+	"os"
```

Comment:
stdlib imports go in the top section, Sia libs go in the middle section, external libs go in the bottom section (including non-Sia NebulousLabs libs)